### PR TITLE
Refactor interface to use React

### PR DESCRIPTION
**Pull request by turnerm**
Resolves issue #26. All the major components are now there, but tagged WIP because the layout and CSS styling still need to be addressed. 

Other issues:
- The interface feels less responsive than the pure d3 version. Furthermore, I put the damage boundary updates to occur only at the end of drag; if they are placed to occur during the drag then the interface kind of freaks out. 
- Related to the above, at the moment the damage category that a particular datapoint belongs to (and hence both the circle and building outline colour) is recalculated in the components that generate the shapes themselves (as opposed to what I believe is the way Marco did it where the data array is modified to assign each point to a category by number). Perhaps by implementing the update the way Marco had it, and then using `shouldComponentUpdate` to only re-render changed points, could help the performance.

### CODE REVIEW

File: caladrius/interface/src/Circles.js [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+
+ componentDidMount() {
+    this.draw_plot()
+  }
+
+ componentDidUpdate() {
+    this.draw_plot()
+  }
+
+  draw_plot() {
```
Comment (turnerm) [inline caladrius/interface/src/Circles.js:41]: verify

File: caladrius/interface/src/Tables.js [hunk 2]
```diff
@@ -0,0 +1,182 @@
...
+    return (
+      <table
+        width={this.props.width}
+        ref={this.ref}
+        />
+    )
+  }
+}
+
+export class PointInfoTable extends Table {
```
Comment (turnerm) [inline caladrius/interface/src/Tables.js]: remove d3 for table stuff

**Review by turnerm — COMMENTED** [review]
Taken care of now:
- adding admin regions to the map
- rendering the tables in html
- not re-drawing the scatterplot points every time, just updating the colours

**Comment by gulfaraz** [comment]
@turnerm I get the following error on running `npm start` from the interface folder,

```
Module not found: Can't resolve './data/Sint-Maarten-2017/admin_regions.geojson' in '.../caladrius_react/caladrius/interface/src'
```

Where can I find `admin_regions.geojson`?

**Comment by turnerm** [comment]
> @turnerm I get the following error on running `npm start` from the interface folder,
> 
> ```
> Module not found: Can't resolve './data/Sint-Maarten-2017/admin_regions.geojson' in '.../caladrius_react/caladrius/interface/src'
> ```
> 
> Where can I find `admin_regions.geojson`?

If you re-run `caladrius_data --create-report-info-file`, it should also output `admin_regions.geojson` (relevant code is [here](https://github.com/rodekruis/caladrius/pull/27/files#diff-95a7a95c35ec7109a2d43b2856f558c7R322)). You will first have to download the admin region files, I've added instructions to the readme [here](https://github.com/rodekruis/caladrius/pull/27/files#diff-04c6e90faac2675aa89e2176d2eec7d8R40).

**Review by gulfaraz — APPROVED** [review]

**Pull request merged**